### PR TITLE
Added (Optional) to optional fields

### PR DESCRIPTION
**Pull request by terechc**
https://github.ibm.com/craig-automation/CRAIG/issues/685

- Added (optional) to the fields in the linked issue. If field was part of IcseTextInput, I added optional as a parameter
### After Review
- Removed (optional) from placeholders and put into label
- Moved all instances of (optional) to the end of the text

### CODE REVIEW

File: src/components/forms/IamAccountSettingsForm.js [hunk 1]
```diff
@@ -183,7 +183,7 @@ class IamAccountSettingsForm extends Component {
           />
           {/* session expiration */}
           <NumberInput
-            placeholder="900"
+            placeholder="(Optional) 900"
```
Comment (jvallexm) [inline src/components/forms/IamAccountSettingsForm.js:186]: let's move `(Optional)` here to label as well

File: src/components/forms/IamAccountSettingsForm.js [hunk 2]
```diff
@@ -204,7 +204,7 @@ class IamAccountSettingsForm extends Component {
           />
           {/* session invalidation */}
           <NumberInput
-            placeholder="900"
+            placeholder="(Optional) 900"
```
Comment (jvallexm) [inline src/components/forms/IamAccountSettingsForm.js:207]: let's move `(Optional)` here to label as well

File: src/components/forms/SysdigForm.js [hunk 3]
```diff
@@ -88,7 +88,7 @@ class SysdigForm extends Component {
             className="fieldWidth"
           />
           <IcseToggle
-            labelText="Platform Logs"
+            labelText="(Optional) Platform Logs"
```
Comment (jvallexm) [inline src/components/forms/SysdigForm.js:91]: let's change `Platform Logs` to `Add Platform Logging`

File: src/components/forms/LogDNAForm.js [hunk 4]
```diff
@@ -112,7 +112,7 @@ class LogDNAForm extends Component {
             className="fieldWidthSmaller"
           />
           <IcseToggle
-            labelText="Platform Logs"
+            labelText="(Optional) Platform Logs"
```
Comment (jvallexm) [inline src/components/forms/LogDNAForm.js:115]: let's change `Platform Logs` to `Add Platform Logging`

File: src/components/forms/LogDNAForm.js [hunk 5]
```diff
@@ -103,7 +103,7 @@ class LogDNAForm extends Component {
               content: "Create an archive with the LogDNA Provider",
               align: "bottom-left",
             }}
-            labelText="Archive"
+            labelText="(Optional) Archive"
```
Comment (jvallexm) [inline src/components/forms/LogDNAForm.js:106]: let's change `Platform Logs` to `Add LogDNA Archive`

File: src/components/forms/VsiLoadBalancerForm.js [hunk 6]
```diff
@@ -412,6 +412,7 @@ class VsiLoadBalancerForm extends React.Component {
           />
           {/* connection limit */}
           <NumberInput
+            placeholder="(Optional)"
```
Comment (jvallexm) [inline src/components/forms/VsiLoadBalancerForm.js:415]: let's move `(Optional)` to Label Text

File: src/components/forms/VsiVolumeForm.js [hunk 7]
```diff
@@ -80,7 +80,7 @@ class VsiVolumeForm extends Component {
             allowEmpty={true}
             step={1}
             hideSteppers={true}
-            placeholder="100"
+            placeholder="(Optional) 100"
```
Comment (jvallexm) [inline src/components/forms/VsiVolumeForm.js:83]: let's move `(Optional)` to Label Text

File: src/components/forms/cbr/CbrRuleForm.js [hunk 8]
```diff
@@ -116,6 +117,7 @@ class CbrRuleForm extends Component {
         </IcseFormGroup>
         <IcseFormGroup>
           <TextArea
+            placeholder="(Optional)"
```
Comment (jvallexm) [inline src/components/forms/cbr/CbrRuleForm.js:120]: let's move `(Optional)` to Label Text

File: src/components/forms/IamAccountSettingsForm.js (hunk 1 above)
Comment (terechc) [reply src/components/forms/IamAccountSettingsForm.js:186]: @jvallexm Do you mean (Optional) should only be in label and not in label and placeholder?  Having both seems like overkill. Wondering for the couple forms you said to move (optional) to label as well
Screenshot 2023-08-15 at 10 26 57 AM

Comment (Chris-Springstead) [reply src/components/forms/IamAccountSettingsForm.js:186]: I think we should follow the format that the IBM Cloud UI follows
image

File: src/components/forms/IamAccountSettingsForm.js [hunk 9]
```diff
@@ -129,7 +129,7 @@ class IamAccountSettingsForm extends Component {
                 "Defines if the entity history is included in the response.",
               align: "top-left",
             }}
-            labelText="Include History"
+            labelText="Include History (optional)"
```
Comment (jvallexm) [inline src/components/forms/IamAccountSettingsForm.js:132]: let's have `(Optional)` at the beginning of labeltext for consistency among forms

**Review by jvallexm — CHANGES_REQUESTED** [review]

**Comment by jvallexm** [comment]
you also need to increase the patch version in package.json by one

**Review by jvallexm — APPROVED** [review]

**Pull request merged**